docs/guides/how_to_guides/validation/how_to_validate_data_without_a_checkpoint.rst

```diff
@@ -3,16 +3,58 @@
 How to validate data without a Checkpoint
 =========================================
 
-.. admonition:: Admonition from Mr. Dickens
+This guide demonstrates how to load an Expectation Suite and validate data without using a :ref:`Checkpoint<how_to_guides__validation__how_to_create_a_new_checkpoint>`. This might be suitable for environments or workflows where a user does not want to or cannot create a Checkpoint.
```

Comment:
This begs the question "When should I use a Checkpoint vs using some other workflow?"

Is there a simple way to answer this in one sentence, or should we write a separate article to cover it?

Reply:
I'd leave explanation out of the how-tos,  but I can say "for example in a hosted environment"?

Reply:
Sure, that makes sense.